app/auth/user.js

```diff
@@ -35,6 +35,18 @@ User.prototype.isFollower = function isFollower(keyInstitution) {
     return isFollower;
 };
 
+User.prototype.isAdmin = function isAdmin(keyInstitution) {
+    var isAdmin = false;
+
+    _.forEach(this.institutions_admin, function(institution) {
```

Comment:
Fica o problema é que this.institutions_admin é uma lista com links assim: 
"http://localhost:8080/api/key/ahJkZXZ-bGl2cmFyaWFiLTEyODByGAsSC0luc3RpdHV0aW9uGICAgICA6L0JDA